packages/dropdowns/examples/advanced-menu.md

```diff
@@ -5,52 +5,138 @@ All `Items` require a `value` prop that is provided with the `onSelect` callback
+        <Dropdown selectedItem={state.placement} onSelect={placement => setState({ placement })}>
+          <Field>
+            <Label>Placement</Label>
+            <Select isCompact>{state.placement}</Select>
+          </Field>
+          <Menu isCompact>
+            <Item value="top">top</Item>
+            <Item value="top-start">top-start</Item>
+            <Item value="top-end">top-end</Item>
+            <Item value="end">end</Item>
+            <Item value="end-top">end-top</Item>
+            <Item value="end-bottom">end-bottom</Item>
+            <Item value="bottom">bottom</Item>
+            <Item value="bottom-start">bottom-start</Item>
+            <Item value="bottom-end">bottom-end</Item>
+            <Item value="start">start</Item>
+            <Item value="start-top">start-top</Item>
+            <Item value="start-bottom">start-bottom</Item>
+          </Menu>
+        </Dropdown>
+        <StyledSpacer />
+        <FormField>
+          <Toggle checked={state.hasArrow} onChange={e => setState({ hasArrow: e.target.checked })}>
+            <FormLabel>Has arrow</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle
+            checked={state.isAnimated}
+            onChange={e => setState({ isAnimated: e.target.checked })}
+          >
+            <FormLabel>Animated</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle
+            checked={state.isCompact}
+            onChange={e => setState({ isCompact: e.target.checked })}
+          >
+            <FormLabel>Compact</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle
+            checked={state.forceIsOpen}
+            onChange={e => setState({ forceIsOpen: e.target.checked })}
+          >
+            <FormLabel>Force open</FormLabel>
+          </Toggle>
+        </FormField>
+      </Well>
+    </Col>
+    <Col size={8} style={{ textAlign: 'center' }}>
+      <Dropdown
+        isOpen={state.forceIsOpen || state.isOpen}
+        onStateChange={changes => {
+          if (Object.prototype.hasOwnProperty.call(changes, 'isOpen')) {
+            setState({ isOpen: changes.isOpen });
+          }
+        }}
+        onSelect={item => alert(item)}
+      >
+        <Trigger>
+          <Button active={state.isOpen}>Advanced Layout</Button>
+        </Trigger>
+        <Menu
+          placement={state.placement}
+          hasArrow={state.hasArrow}
+          isCompact={state.isCompact}
+          isAnimated={state.isAnimated}
+        >
+          <HeaderItem containsIcon>
+            <HeaderIcon>
+              <ClipboardIcon />
+            </HeaderIcon>
+            Header Item
+          </HeaderItem>
+          <Item value="profile">Option 1</Item>
+          <Item value="settings">Option 2</Item>
+          <Item disabled>Disabled item</Item>
```

Comment:
Recommend removing this `disabled` item and adding a "Disable items" toggle to the controls. Two reasons:
1) it lets us vet that disabled styling works as intended across all of the item types
2) it removes a wonky visual anti-pattern where the first view of the menu plants a `separator` directly in line with the arrow

![Screen Shot 2020-01-29 at 2 03 29 PM](https://user-images.githubusercontent.com/143773/73401555-346d4b80-42a0-11ea-9962-e5fe164f70b8.png)

Reply:
Good points. I've added a disabled option to the knobs.